examples/http_mitm_proxy_boring.rs

```diff
@@ -80,6 +77,7 @@ use rama::{
         profile::UserAgentDatabase,
     },
 };
+use rama_tls_boring::client::TlsConnectorDataBuilder;
```

Comment:
tooling sadly doesn't help here, but please make sure to double check these, especially in outside examples but also in actual lib code. Make sure to use the appropriate re-exports when available.

These `/examples` should be copy paste-able for someone who _only_ has `rama` as its dep and if needed also some third-party dep, but certainly no other rama crate.